### PR TITLE
Make AC motor check more region agnostic

### PR DESCRIPTION
**Pull request by GingerSnap-xx**
Starting to get myself familiar with the codebase to try to resolve a bug I reported. I noticed this voltage check seems to be assuming an EU or UK mains voltage of 220. In the US it is 120 (or 110 depending on who you ask), and according to Wikipedia, some countries (Japan) go as low as 100!

Without some knowledge that regardless of what the true ac voltage is it gets reported as 220, I assumed this was something that might not be accurate in all cases and figured I'd submit a fix to open it up a bit.

### CODE REVIEW

File: motionblinds/motion_blinds.py [hunk 1]
```diff
@@ -1051,8 +1051,11 @@ def _calculate_battery_level(self, voltage):
             # 4 cel battery pack (16.8V)
             return round((voltage - 14.6) * 100 / (16.8 - 14.6), 0)
 
-        if voltage == 220.0:
+        if voltage >= 100.0:
             # AC motor
+            #There are a myriad of residential mains voltages, 
+            # but all of them seem to be over 100.
+            #https://en.wikipedia.org/wiki/Mains_electricity_by_country
```
Comment (starkillerOG) [inline motionblinds/motion_blinds.py:1058]:
```suggestion
```

**Review by starkillerOG — APPROVED** [review]
LGTM, I removed the comment, because I don't think it is needed

**Comment by starkillerOG** [comment]
Thank you for the PR, did you actually have a AC blind reporting a voltage between 100 and 220 V?

**Comment by GingerSnap-xx** [comment]
> Thank you for the PR, did you actually have a AC blind reporting a voltage between 100 and 220 V?

I did not have one myself, so it was mostly speculative.  The only reason I would think you wouldn't get 120 would be is if there's a transformer at play upstream of the sensor. I can try to get a hold of a model that uses AC motors to test it. If  you already know of a good low cost one I could use that'd be awesome otherwise I'll look around.

**Comment by starkillerOG** [comment]
I don't think testing will be nessesary.
Besides it could very well be that the voltage is hardcoded to 230V and not actually measured. (Why would they measure the 230V, kind of pointless)

**Pull request merged**